### ISSUE TITLE
Import fails on windows 10

### ISSUE THREAD

**Issue by croots** [btk5h/skript-mirror#153]
**Describe the bug**
When attempting to import class files, skript throws exception.

**To reproduce**
Rename attached skript to filetype .sk and run in
 a windows environment.

**Expected behavior**
Import behaves cleanly as it does on linux-based OS's

**Server information**
- skript-mirror: 1.0.0 & 0.19.1
- Skript: 2.4.1
- Bukkit: Paper 283
- Minecraft: 1.15.2
- Java: Build 1.8.0_251-b08
- OS: Windows Version 10.0.18363 Build 18363

**Additional context**
Thrown error attached.

All scripts seem to work fine after /skript reload all, issue only seems to occur for first script alphabetically. Perhaps Skript is trying to load scripts on windows before skript-mirror is totally initialized? 

[errortestskript.txt](https://github.com/btk5h/skript-mirror/files/4629955/errortestskript.txt)

[error.log](https://github.com/btk5h/skript-mirror/files/4629953/error.log)

**Comment by croots** [comment]
Appears to be fixed in 2.0.0 snapshot? I don't see anything about it in the issue tracker so it might be useful to add a check somewhere if the fix was unintentional to prevent a future update from breaking again.

**Comment by btk5h** [comment]
Yes, I believe this was not an OS-specific issue, but rather an issue with how 1.X hacks around Skript's logger to disable certain warnings. In recent versions of Skript, Skript's logger implementation changed, breaking this hack.

This logger hack has been removed in the 2.X branch (and there are no plans to add it back), but it's worth backporting the fix to 1.X as well. I'll try to fix that this weekend, assuming the fix is relatively quick to make.
